src/_P078_Eastron.ino

```diff
@@ -3,13 +3,13 @@
 #ifdef USES_P078
 
 // #######################################################################################################
-// ############## Plugin 078: SDM120/SDM120CT/220/230/630/72D/DDM18SD Eastron Energy Meter ###############
+// ############## Plugin 078: SDM120/SDM120CT/220/230/630/72D/DDM18SD/TAC2100 Eastron Energy Meter ###############
 // #######################################################################################################
 
 /*
    Plugin written by: Sergio Faustino sjfaustino__AT__gmail.com
 
-   This plugin reads available values of an Eastron SDM120C SDM120/SDM120CT/220/230/630/72D & also DDM18SD.
+   This plugin reads available values of an Eastron SDM120C SDM120/SDM120CT/220/230/630/72D & also DDM18SD and TAC2100.
```

Comment:
Can you also add a line to the Changelog, below? Newest entry on top, like the entries I already added.